packages/vuetify/src/components/VField/VField.sass

```diff
@@ -82,7 +82,7 @@
     @at-root
       @include tools.density('v-input', $input-density) using ($modifier)
         @at-root #{selector.nest(&, $root)}
-          --v-field-padding-bottom: #{15px + $modifier * .5}
+          --v-field-padding-bottom: #{16px + $modifier * .5}
```

Comment:
Default density seems slightly off center, should this still be 15?

Reply:
Before the input was assigned uneven chips margin top/bottom, in order to be aligned with chips, so it looks bit off center.

Now css gap is being used and chips margin top/bottom are completely removed.

Input/chips should be always aligned center in outlinded variant or without labels.